src/indexer/repository/transfer.ts

```diff
@@ -75,6 +79,23 @@ class TransferRepository {
     return await this.transfer.create({ data: transferData })
   }
 
+  public async insertExecutionSubstrateTransfer(originDomainId: string, depositNonce: number): Promise<Transfer> {
```

Comment:
Instead of making separate functions for each network accept execution params and have one function ınsertExecutionTransfer` (remove decodedLog from insertExecutionTransfer)

Reply:
done